packages/angular/projects/cds-angular/package.json

```
@@ -8,8 +8,8 @@
     "url": "https://github.com/vmware/clarity.git"
   },
   "peerDependencies": {
-    "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0",
+    "@angular/common": "^11.0.0",
```

That way, installing the package in a project that uses Angular 9 or 10 will show a warning that `@cds/angular` requires Angular 11. Shouldn't the package be fine with these versions?

---

Yeah, good catch. I'll get it updated for `>9`

---

Yes, if we want it to respect 9, 10 and 11, we should set it to
```suggestion
    "@angular/common": "9.x || 10.x || 11.x",
```